package.json

```diff
@@ -4,14 +4,21 @@
   "description": "E4P JumboCode Project",
   "main": "server.js",
   "scripts": {
+    "lint-all": "npm run lint-client && npm run slint-client && npm run hlint-client",
     "test": "mocha --exit --recursive tests",
     "start": "node server.js",
     "slint": "stylelint",
-    "slint-client": "stylelint ./public/**/*.css"
+    "slint-client": "stylelint ./public/**/*.css",
+    "hlint": "htmlhint",
+    "hlint-client": "htmlhint ./public/**/*.html",
+    "lint": "eslint",
+    "lint-client": "eslint ./public/**/*.js",
+    "lint-server": "eslint server.js",
+    "lint-tests": "eslint ./tests/**/*.js"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run slint-client"
+      "pre-commit": "npm run lint-all"
```

Comment:
I'm not sure if we want to do this yet. Other people are still working on branches with significant style differences. I think I'd rather have just a simple style lint required for commits so that this doesn't prevent anyone from contributing. We can then have a transition period where we show team members how to use the linter, configure their IDE and such.